tests/mock_crust/churn.rs

```diff
@@ -90,6 +92,44 @@ fn did_receive_get_success(node: &TestNode,
     }
 }
 
+fn quorum_before(nodes: &[TestNode],
+                 src: Authority,
+                 dst: Authority,
+                 data_id: DataIdentifier,
+                 message_id: MessageId)
+                 -> usize {
+    let group = nodes.iter()
+        .filter(|n| n.routing_table().is_recipient(&src.to_destination()))
```

Comment:
That should be the destination, shouldn't it?
If I remember correctly, the problem was that the _destination_ group had a different size before and after the churn, and we needed to take both into account.